v1/formats/bdiv.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>Bangladeshi Asian Parlimentary</name>
+  <version>1</version>
+  <info>
+    <region>Bangladesh</region>
+    <level>University</level>
+    <used-at>Bangladesh National Intervarsity</used-at>
+    <description>3 vs 3, 5-minute speeches, 3-minute replies, POIs/POOs/POPs allowed</description>
+  </info>
+  <prep-time length="15:00"/>
+  <speech-types>
+    <speech-type ref="substantive" length="5:00" first-period="normal">
+      <bell time="1:00" number="1" next-period="pois-allowed"/>
+      <bell time="4:00" number="2" next-period="warning"/>
+      <bell time="finish" number="3" next-period="overtime"/>
+    </speech-type>
+    <speech-type ref="reply" length="4:00" first-period="normal">
```

Comment:
The description says that the replies are 3 minutes, but the speech-type specification says 4 minutes. Which is it?